lib/vfscore/Config.uk

```diff
@@ -785,4 +786,8 @@ if LIBVFSCORE_AUTOMOUNT_EINITRD
 		image and mounted under '/'.
 endif
 
+config LIBVFSCORE_AUTOUNMOUNT
```

Comment:
> Only in the case that automatic mounting is enabled, the unmounting option is activated.

But it looks like it could be selected individually? Also, I guess that if a user disables automounting in favor of having his app do the mounting, they would have to know to also enable this config right?

Reply:
The idea is that a user could program its own mounting logic whil just take this option for letting handle the unmount automatically. If you slect automount in vfscore, it will enable also unmounting.